server/api.js

```diff
@@ -134,4 +135,33 @@ router.post("/publish-tenders", (req, res) => {
 	res.status(200).json({ message: "Form submitted successfully!" });
 });
 
+router.get("/tenders", async (req, res) => {
+	const { page = 1, limit = 25 } = req.query;
+	const offset = (page - 1) * limit;
+	try {
+		const result = await query(
+			`
+		SELECT
+			t.id AS "Tender ID",
+			t.title AS "Tender Title",
+			t.creation_date AS "Tender Created Date",
+			t.announcement_date AS "Tender Announcement Date",
+			t.deadline AS "Tender Closing Date",
+			t.deadline AS "Tender Project Deadline Date",
+			t.status AS "Tender Status"
```

Comment:
I don't think it's a good idea to create these aliases. Let the client decide the headers to user for the table

Reply:
I have modified it